qiskit/releasenotes/notes/bip-improvement-34cd623294a30d83.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    The ``balanced`` objective function of the ``BIPMapping`` no longer
+    requires a ``BackendProperties``. It will use ``BackendProperties`` if
+    available, otherwise it assumes a CX error rate as given in the new
+    parameter ``default_cx_error_rate``. A new parameter ``depth_obj_weight``
+    now controls the weight of the two objectives combined in the ``balanced``
+    objective.
+other:
```

Comment:
may be `upgrade` because this may requires end users to change their code.

Reply:
I changed to 'upgrade' in the next commit.